app/main.py

```diff
@@ -1,8 +1,35 @@
 class Person:
-    # write your code here
-    pass
+
+    people: dict = {}
```

Comment:
It is not a good practice to use a mutable default argument (`people: dict = {}`) in the class definition. Mutable default arguments in Python can lead to unexpected behavior, especially when dealing with multiple instances of the class. It is recommended to initialize such attributes in the constructor (`__init__`) method instead.